.github/workflows/ci.yml

```diff
@@ -29,20 +29,48 @@ jobs:
       - run: ${{ matrix.lint-command }}
 
   tests:
-    name: Python ${{ matrix.python-version }}
+    name: Python ${{ matrix.python-version }}, Django ${{ matrix.django-version }}, with ${{ matrix.database }}
+    needs: linters
```

Comment:
We can remove this line. This makes the tests matrix wait for  the linters one to finish before start. We can leave both of them running in parallel if we remove this clause.

```suggestion
```